kettle/update.py

```diff
@@ -16,13 +16,32 @@
 
 
 import os
+from datetime import datetime
 
+DUMP = 'dump.txt'
 
-def call(cmd):
-    print('+', cmd)
-    status = os.system(cmd)
+def print_dump(file):
+    if os.path.exists(file):
+        with open(file, "r") as fp:
+            output = fp.read()
+        os.remove(file)
+        print(output)
+    else:
+        print(f'unable to find dump file: {file}')
+
+
+def call(cmd, dump=False):
+    started = datetime.now()
+
+    redirect = ' > {DUMP}}' if dump else ''
+    status = os.system(f'{cmd}{redirect}')
```

Comment:
I don't think literal expansion happens recursively?
```
>>> foo = 'monkeys'
>>> bar = '{foo}'
>>> print(f'{bar}')
{foo}
```
Reassigning to cmd means we can include the redirect in further logging
```suggestion
    cmd = f'{cmd} > {DUMP}' if dump else cmd
    status = os.system(cmd)
```

Reply:
Good catch